CIScripts/Common/Invoke-UntilSucceeds.ps1

```diff
@@ -0,0 +1,56 @@
+. $PSScriptRoot\Aliases.ps1
+. $PSScriptRoot\Exceptions.ps1
+
+function Invoke-UntilSucceeds {
+    <#
+    .SYNOPSIS
+    Utility wrapper for making sure that the assert is consistently true. It works by retrying
+    the assert every Interval seconds, up to Duration.
```

Comment:
This description doesn't match the function name.

Reply:
Also, this function tries to do two things at once - checking return value and looking at exceptions. Perhaps split this functionality or update the description and function name?

Reply:
This function was designed to check return val and catch exception. `Invoke-UntilSucceeds` was a name that we came up with @PawelMarchewkaCodiLime. Updated the description though.